src/metabase/cloud_migration/models/cloud_migration.clj

```diff
@@ -235,7 +245,7 @@
           (do
             (t2/update! :model/CloudMigration id {:state :error})
             (log/info "Migration failed")
-            (throw (ex-info "Error performing migration" {:error e})))))
+            (throw (ex-info "Error performing migration" {} e)))))
```

Comment:
Can revert this if this breaks something, but wanted to set the cause rather than packing an exception into the arbitrary data map.